test/eventlistener_scale_test.go

```diff
@@ -47,8 +47,22 @@ func TestEventListenerScale(t *testing.T) {
 	// Create an EventListener with 1000 Triggers
 	var err error
 	el := bldr.EventListener("my-eventlistener", namespace, bldr.EventListenerSpec(
-		bldr.EventListenerServiceAccount(saName)),
-	)
+		bldr.EventListenerServiceAccount(saName),
+		bldr.EventListenerPodTemplate(
+			bldr.EventListenerPodTemplateSpec(
+				bldr.EventListenerPodTemplateTolerations(nil),
+				// If you want to test toleration feature, please use this commented part.
```

Comment:
Is the test commented out because it will fail in our CI runs?

Reply:
No.. it worked even though I uncommented it. I commented out those lines because I think test process cannot verify it really works because I guess there is no tainted node for e2e test. I uncommented every lines about tolerations in test files.